src/python/tools/Report_NonAlterableUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,97 @@
+http://projecteuclid.org/DPubS?service=UI&version=1.0&verb=Display&handle=euclid.aos/1013203451,.\src\python\docs\docstrings\FastTreesTweedieRegressor.txt
+http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.352.7619,.\src\python\docs\docstrings\GamBinaryClassifier.txt
+http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.352.7619,.\src\python\docs\docstrings\GamBinaryClassifier.txt
+http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.352.7619,.\src\python\docs\docstrings\GamRegressor.txt
+http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.352.7619,.\src\python\docs\docstrings\GamRegressor.txt
+http://arxiv.org/abs/1412.1576,.\src\python\docs\docstrings\LightLda.txt
+http://arxiv.org/abs/1412.1576,.\src\python\docs\docstrings\LightLda.txt
+http://research.microsoft.com/en-,.\src\python\docs\docstrings\LocalDeepSvmBinaryClassifier.txt
+http://research.microsoft.com/en-,.\src\python\docs\docstrings\LocalDeepSvmBinaryClassifier.txt
+http://en.wikipedia.org/wiki/L-BFGS,.\src\python\docs\docstrings\LogisticRegressionBinaryClassifier.txt
+http://en.wikipedia.org/wiki/Logistic_regression,.\src\python\docs\docstrings\LogisticRegressionBinaryClassifier.txt
+http://research.microsoft.com/apps/pubs/default.aspx?id=78900,.\src\python\docs\docstrings\LogisticRegressionBinaryClassifier.txt
+http://en.wikipedia.org/wiki/L-BFGS,.\src\python\docs\docstrings\LogisticRegressionClassifier.txt
+http://en.wikipedia.org/wiki/Logistic_regression,.\src\python\docs\docstrings\LogisticRegressionClassifier.txt
+http://research.microsoft.com/apps/pubs/default.aspx?id=78900,.\src\python\docs\docstrings\LogisticRegressionClassifier.txt
+http://research.microsoft.com/pubs/69731/tr-99-87.pdf,.\src\python\docs\docstrings\OneClassSVMAnomalyDetector.txt
+http://www.stat.purdue.edu/~yuzhu/stat598m3/Papers/NewSVM.pdf,.\src\python\docs\docstrings\OneClassSVMAnomalyDetector.txt
+http://web.stanford.edu/group/mmds/slides2010/Martinsson.pdf,.\src\python\docs\docstrings\PcaAnomalyDetector.txt
+http://arxiv.org/pdf/1106.5730v2.pdf,.\src\python\docs\docstrings\SgdBinaryClassifier.txt
+http://arxiv.org/pdf/1106.5730v2.pdf,.\src\python\docs\docstrings\SgdBinaryClassifier.txt
+http://arxiv.org/pdf/1106.5730v2.pdf,.\src\python\docs\docstrings\SgdBinaryClassifier.txt
+http://arxiv.org/pdf/1106.5730v2.pdf,.\src\python\docs\docstrings\SgdBinaryClassifier.txt
+http://arxiv.org/pdf/1206.6910.pdf,.\src\python\docs\docstrings\SsaForecaster.txt
+http://anthology.aclweb.org/P/P14/P14-1146.pdf,.\src\python\docs\docstrings\SsweEmbedding.txt
+http://anthology.aclweb.org/P/P14/P14-1146.pdf,.\src\python\docs\docstrings\SsweEmbedding.txt
+http://www.aaai.org/Papers/KDD/1996/KDD96-019.pdf,.\src\python\docs\docstrings\SupervisedBinner.txt
+http://anthology.aclweb.org/P/P14/P14-1146.pdf,.\src\python\docs\docstrings\WordEmbedding.txt
+http://matplotlib.org,.\src\python\docs\sphinx\conf.py
```

Comment:
>http://matplotlib.org [](start = 0, length = 21)

Why this is not alterable? [https://matplotlib.org](https://matplotlib.org) is valid